test/WhiteBoxTests.cpp

```diff
@@ -963,8 +969,16 @@ void WhiteBoxTests::testTime()
 
         const std::string s = Util::getIso8601FracformatTime(t);
         t = Util::iso8601ToTimestamp(s, "LastModifiedTime");
-        LOK_ASSERT_EQUAL(std::to_string(t_in_micros),
-                             std::to_string(t.time_since_epoch().count()));
+
+        std::string t_in_micros_str = std::to_string(t_in_micros);
```

Comment:
Would have been good to add examples of the values these strings might have in a comment, so it would be easier to follow what's going on. 

My guess is that on some systems there is no nano-second precision (microseconds only), but it's not clear why we are trimming the last 3 digits from *both* values. I assume it's because one has "000" at the end, and the other has some non-zero three digits. Again, I'm guessing and can't validate (because on my system I have nanosecond precision), hence my suggestion for a comment with explanation/example.